client/lxd_storage_volumes.go

```diff
@@ -943,6 +943,63 @@ func (r *ProtocolLXD) GetStoragePoolVolumeBackupFile(pool string, volName string
 	return &resp, nil
 }
 
+// CreateStoragePoolVolumeFromISO creates a custom volume from an ISO file.
+func (r *ProtocolLXD) CreateStoragePoolVolumeFromISO(pool string, args StoragePoolVolumeBackupArgs) (Operation, error) {
+	if !r.HasExtension("custom_volume_iso") {
```

Comment:
We should use r.CheckExtension() nowadays to avoid duplicating the (and risking a copy paste error when updating it in both places) the extension name.

Reply:
Nice. I didn't know about that function.